src/modules/ChannelList/dux/__tests__/reducers.spec.js

```diff
@@ -906,4 +919,51 @@ describe('Channels-Reducers', () => {
       expect(hideWithPreventAutoOnPreventAutoState.allChannels.length).toEqual(preventAutoUnhideParamsState.allChannels.length + 1);
     });
   });
+  describe.only('isChannelJustCreated', () => {
```

Comment:
need to remove `only`

Reply:
https://github.com/sendbird/sendbird-uikit-react/pull/1124/commits/d691222b5b7c6e7f4cd78767cceb1a13579c972f#diff-9536f4716bc6385ea2e3e35374ac6abb3ec4259a4881777212f637e53f1fce71L922 #449